action.yml

```diff
@@ -38,12 +38,12 @@ runs:
         if [ -z "$IMAGE" ]; then
           IMAGE="https://github.com/$ACTION_REPO/releases/download/$ACTION_REF/images.tar.xz"
         fi
-        if [ -f "$IMAGE" ] && [ $(realpath "$IMAGE") == "$(realpath ${{github.action_path}}/images.tar.xz)" ]; then
+        if [ -f "$IMAGE" ] && [ $(realpath "$IMAGE") == "$(realpath ${{ github.action_path }}/images.tar.xz)" ]; then
           echo File already exists;
         elif [ -f "$IMAGE" ]; then
-          cp "$IMAGE" ${{github.action_path}}/images.tar.xz;
+          cp "$IMAGE" ${{ github.action_path }}/images.tar.xz;
         else
-          wget -q --no-verbose "$IMAGE" -O ${{github.action_path}}/images.tar.xz;
+          wget -q --no-verbose "$IMAGE" -O ${{ github.action_path }}/images.tar.xz;
```

Comment:
So does `${{ github.action_path }}` work here? Or should all its occurrences be changed to `$GITHUB_ACTION_PATH`? I've not used GitHub Actions containers, but I think it should be the latter? Though I guess I shouldn't expect any GHA feature to be intuitive :)